.github/workflows/bump_release_version.yml

```diff
@@ -0,0 +1,35 @@
+#############################################################################
+# GitHub Action to bump release version
+#
+#############################################################################
+name: "Bump version"
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  bump:
+    name: Bump version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Modify File
+        run: |
+          RELEASE_VERSION=$(git describe --abbrev=0 | cut -d '-' -f 2)
+          ./.ci/bump-release-version.sh "$RELEASE_VERSION"
+      - name: GitHub Milestone
+        run: |
+          .ci/update-github-milestone.sh "$RELEASE_VERSION"
+      - name: Push commit
```

Comment:
Put push before update of milestone, if any problem with push happens better to not change milestone

Reply:
Changes Done